### PR TITLE
Added 4 BS multi-select widgets to indicator form.

### LINKED ISSUE #1023: Disaggregation/Indicator Type selector function is confusing
**Description:**
On the Performance tab when creating a new or editing an indicator, it is difficult to understand the disaggregation and indicator type fields based on how a selection can be made. 
Though there is no indication that these two options with selections are required, there is no way to DE-select an option once clicked. 

**Steps to Reproduce:**
1. Login to Tola-Activity
2. Click Testing specific Program "TolaData Test"
3. Click "+ Add Indicator"
4. Add name to Indicator and click "Save Changes"
5. Click the performance tab
6. Click/Highlight options under disaggregation & indicator type

**Results:**
Multiple options can be "selected" by clicking and dragging for highlighting. When clicking an option, there is no way to deselect an option. 

**Expected Results:**
If multiple selection is allowed, check box or similar selector is used. If not an optional field/selection, mark properly and block saving as required. 

**Attachments:**
screen shot 2019-01-04 at 9 22 40 am

### PR DESCRIPTION
**Pull request by KennyMonster**
Removed select2 usage on Country Strategic Objective

Downward triangle on multiselect widget still vertically aligned.

Closes #1023

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
         <!-- Hosted Leaflet -->
         <script src="{% static "js/leaflet.js" %}"></script>
 
+        <!-- bootstrap multiselect -->
+        <script type="text/javascript" src="{% static "js/bootstrap-multiselect.js" %}?v=w"></script>
```
Comment (axoplasm) [inline templates/base.html]: Why did you move this to the top of the doc?

Comment (KennyMonster) [reply templates/base.html]: On the full page version of the form, the helpers include is included in the "content" block, which is loaded before the old script tag at the bottom of the page. Thus `.multiselect()` failed because the lib had not been loaded yet.

The old code worked around this by included the script tag in the content body redundantly when using select2. This seems kinda jank.

I realize having more JS in the head is not ideal for load/render times, but I can't think of a better workaround with the current system of nested includes/blocks + script tags positioned right next to HTML.
